### PR TITLE
First pass at adding route wiring for a useful route

### PR DESCRIPTION
**Pull request by NoahGS**

### CODE REVIEW

File: app/api/v1/predict_construction_time/app.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    def get(self):
+        """
+        Return the list of all applicable model versions, optionally with their creation dates?
+        :return: Map<model_version_label, date_created>
+        """
+        #  DO WORK HERE
+        pass
+
+
+@ns.route(process_route + '/<int:model_ref>/cnc_quick_materials')
```
Comment (bwest-at-xometry) [inline app/api/v1/predict_construction_time/app.py]: There is very likely a `prefix` parameter to the `route` decorator.

Comment (bwest-at-xometry) [reply app/api/v1/predict_construction_time/app.py]: but if we were shooting for the convention

`predict/v1/modelN/api`

`predict` and `v1` would get added later to the mix.

Like during namespace creation or at add_namespace time.

Comment (NoahGS) [reply app/api/v1/predict_construction_time/app.py]: I think the bit I'm struggling with is trying to compose the Namespace elements at multiple layers - we want the top-level {predict}, the per-version {v1}, and then the parameterized {model#} and api will probably drop in together. I _may_ have a way to deliver on that, but it doesn't feel terribly pretty.
I'll take a stab at getting the concept in my head implemented (not prettily), and then clean it up from there.

File: app/app.py [hunk 2]
```diff
@@ -88,23 +88,23 @@ def echo():
 # /echo_plus endpoint
 ###############################################################################
 
-api = flask_restplus.Api(app)
+api_instance = flask_restplus.Api(app)
```
Comment (bwest-at-xometry) [inline app/app.py:91]: add `endpoint` parameter to API constructor:
```
api_instance = flask_restplus.Api(app, endpoint='/predict')
```
and where we add the namespace to the Api
```
api.add_namespace(ns, path='/v1')
```
Hopefully this implies that the route for
```
@ns.route('/do_something')
def do_something(): ...
```
would aggregate to `/predict/v1/do_something` (but can not rule out `/v1/predict/do_something`).

I'm 51% sure.

**Pull request closed without merging**